Modules/_json.c

```diff
@@ -1356,6 +1356,27 @@ _steal_accumulate(_PyUnicodeWriter *writer, PyObject *stolen)
     return rval;
 }
 
+
+PyObject* _create_newline_indent(PyObject* indent, Py_ssize_t indent_level)
```

Comment:
```suggestion
PyObject *_create_newline_indent(PyObject *indent, Py_ssize_t indent_level)
```
(And many more times below.) Both are valid syntax in C but the convention in this file is to put `*` after the space.

Reply:
Add `static` and start the function name from a new line:
```suggestion
static PyObject *
_create_newline_indent(PyObject *indent, Py_ssize_t indent_level)
```